pom.xml

```diff
@@ -3422,6 +3428,10 @@
         <version>4.2.0</version>
         <extensions>true</extensions>
       </plugin>
+      <plugin>
```

Comment:
Do more tests,  It's OK to configure it only in `pluginManagement`. Will remove this.